tests/main/uc20-create-partitions-reinstall/task.yaml

```diff
@@ -45,7 +51,8 @@ execute: |
     udevadm info --query=property "${LOOP}p2" | grep ID_FS_TYPE ||:
 
     echo "Run the snap-bootstrap tool in auto-detect mode"
-    /usr/lib/snapd/snap-bootstrap create-partitions ./gadget-dir
+    go get ../../lib/create-partitions
```

Comment:
Perhaps we should tread it like `gendeveloper1model` and `go get` the binary in prepare.sh.